.pre-commit-config.yaml

```diff
@@ -36,3 +36,47 @@ repos:
           - types-aiobotocore[essential]
           - boto3-stubs[essential]
         exclude: ^(diracx-client/src/diracx/client/_generated|diracx-[a-z]+/tests/|diracx-testing/|build|extensions/gubbins/gubbins-client/src/gubbins/client/_generated)
+
+-   repo: https://github.com/codespell-project/codespell
+    rev: v2.4.1
+    hooks:
+    -   id: codespell
+        args: ["--skip=diracx-client/src/diracx/client/_generated/*, diracx-[a-z]*/tests/*, diracx-testing/*, build/*, extensions/gubbins/gubbins-client/src/gubbins/client/_generated/*"]
+
+-   repo: https://github.com/executablebooks/mdformat
+    rev: 0.7.22  # Use the ref you want to point at
+    hooks:
+    -   id: mdformat
+      # Optionally add plugins
+        additional_dependencies:
+        -   mdformat-gfm
+        -   mdformat-black
+        exclude: ^(diracx-client/src/diracx/client/_generated|diracx-[a-z]+/tests/|diracx-testing/|build|extensions/gubbins/gubbins-client/src/gubbins/client/_generated)
+
+-   repo: https://github.com/pre-commit/pygrep-hooks
+    rev: v1.10.0  # Use the ref you want to point at
+    hooks:
+    -   id: python-use-type-annotations
+    -   id: python-check-blanket-noqa
+    -   id: python-check-blanket-type-ignore
+    -   id: python-no-eval
+    -   id: python-no-log-warn
+    -   id: rst-backticks
+    -   id: rst-directive-colons
+    -   id: rst-inline-touching-normal
+    -   id: text-unicode-replacement-char
+        exclude: ^(diracx-client/src/diracx/client/_generated|diracx-[a-z]+/tests/|diracx-testing/|build|extensions/gubbins/gubbins-client/src/gubbins/client/_generated)
+    # ...
+
+-   repo: https://github.com/adamchainz/blacken-docs
+    rev: "v1.12.1"  # replace with latest tag on GitHub
```

Comment:
dependabot will pump it up

Reply:
Do you mean, the version here (I copied it from GitHub) is not relevant ?

Reply:
Leave whatever version now, remove the comment